src/Homely.AspNetCore.Hosting.CoreApp/Program.cs

```diff
@@ -5,11 +5,33 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
 using Serilog;
+using Serilog.Core;
 
 namespace Homely.AspNetCore.Hosting.CoreApp
 {
     public static class Program
     {
+        private static string Explosion = @"" + Environment.NewLine +

```

Comment:
i don't get it.. this was already merged in the [other PR]?(https://github.com/Homely/Homely.AspNetCore.Hosting.CoreApp/pull/13/files)

Reply:
AH! i didn't change the branch on mine :/